packages/eslint/rollup.config.mjs

```diff
@@ -1,7 +1,13 @@
 import { readFileSync } from 'fs';
 
+import esmShim from '@rollup/plugin-esm-shim';
+import typescript from '@rollup/plugin-typescript';
```

Comment:
why do we need these plugins now?

Reply:
eslint plugin use `require` [here](https://github.com/rollup/plugins/blob/8550c4b1925b246adbd3af48ed0e5f74f822c951/packages/eslint/src/index.ts#L17). So we may need a esm-shim plugin to support using `require` in esm bundle?
using typescript plugin here is just for providing the default config in `createConfig()` function

Reply:
you're right, we don't need esm-shim for it.